cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,235 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-logging"]
+= Tutorial: Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-logging
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-05-01
+
+There are various methods to view your logs in {product-rosa} (ROSA). Use the following procedures to forward the logs to AWS CloudWatch, view the logs directly through the pod by using `oc logs`, and use LokiStack to search the logs.
```

Comment:
LokiStack is not used to view the logs.  If we decide to keep the oncluster logging solution then it would be through observability in the web console.